index.html

```diff
@@ -12,6 +12,52 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <img src="img/karma-logo.svg" alt="karma logo" class="logo">
+            <ul class="navbar">
+                <li><a href="#">Meet Karma </a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>   
+    </header>
+
+    <main>
+        <section class="main">
+           <h1>Introducing Karma</h1>
+           <h2>Bring WIFI with you, everywhere you go</h2>  
+                <div class="button">
+                    <p>Learn More</p>
+                </div>    
+        </section>
+        <p class ="tagline">Everyone needs a little Karma</p>
+        <container class="mobile-view">
+            <div class="options">
+                <img src="img/icon-devices.svg" alt="devices" class="devices">
+                <img src="img/icon-coffee.svg" alt="coffee cup" class="coffee-cup">
+                <img src="img/icon-refill.svg" alt="refill icon" class="refill">
+            </div>
+                <div class="text">
+                    <p class="devices-text">Internet for all devices</p>
+                    <p class="coffee-text">Boost your productivity</p>
+                    <p class="refill-text">Pay as you go</p>
+                </div>
```

Comment:
These elements should be grouped differently so that each image 'lives' with its text in the same container. Some like this, but 3 times: 
```
                <div class="icon-wrapper">
                    <img src="img/icon-devices.svg" alt="devices" class="devices">
                    <p class="devices-text">Internet for all devices</p>
                </div>
```
This will ensure that in mobile mode, when they are stacked, the images are still next to the corresponding text.